diff --git a/CMakeLists.txt b/CMakeLists.txt
index 78016a9..7c685f0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -236,7 +236,9 @@ if (FMT_INSTALL)
 
   # Install the library and headers.
   install(TARGETS ${INSTALL_TARGETS} EXPORT ${targets_export_name}
-          DESTINATION ${FMT_LIB_DIR})
+	  RUNTIME DESTINATION bin
+	  LIBRARY DESTINATION ${FMT_LIB_DIR}
+	  ARCHIVE DESTINATION ${FMT_LIB_DIR})
   install(FILES ${FMT_HEADERS} DESTINATION ${FMT_INC_DIR})
 endif ()
 
